hnginternship5/ziki
# Created breathing space for the navbar icon on mobile

#### What does this PR do?
* It creates space at the top of the page for the navbar
#### How should this be manually tested?
* Open the site on your browser
* Type localhost/published-posts
* You should see the navbar icon spaced from the main content

#### Screenshots (if appropriate)
![Screen Shot 2019-04-21 at 08 40 57](https://user-images.githubusercontent.com/33119163/56467081-f2aa9d00-6411-11e9-8524-6d7cb6d49143.png)
